src/IIIFPresentation/Core/Web/HttpRequestX.cs

```diff
@@ -24,6 +26,10 @@ public static string GetDisplayUrl(this HttpRequest request, string? path = null
         var queryString = includeQueryParams
             ? request.QueryString.Value ?? string.Empty
             : string.Empty;
+        if (!string.IsNullOrEmpty(path) && !path.StartsWith(PathDelimiter))
```

Comment:
See comment in `TypedPathTemplateOptions` - if we want paths to start with `/` then the templates should.